### ISSUE TITLE
Map Icons in Classic Era

### ISSUE THREAD

**Issue by Gogo1951** [Questie/Questie#6747]
### Description

When I have more than 1 quest for a dungeon where the dungeon also has a quest in it, the icons have a weird box around them.

<img width="237" alt="image" src="https://github.com/user-attachments/assets/6c08e91d-e2d5-4e20-888f-1db6c8c30720">

<img width="203" alt="image" src="https://github.com/user-attachments/assets/fc8bca09-7ba6-4647-8b06-25233eff1d6e">

Happens in map and minimap. 

### Questie version

Latest

### Game flavor

Era/HC/Anniversary

**Comment by Gogo1951** [comment]
Happens consistently when I reload. 

(Also, I reload before reporting any bug now. Ha.)

**Comment by Gogo1951** [comment]
Hey, just wanted to clarify. It's not a "glow" -- this is like a broken image, or somehow the part of the image that is supposed to be transparent is turning red instead.

**Comment by BreakBB** [comment]
Just to make sure (since you didn't answer that): You disabled all addons except Questie and the behavior is the same?

**Comment by BreakBB** [comment]
Coincidence, but I also got a character and noticed that the glow is looking kind of weird. I am quite sure it's the same issue and actually the glow that is bugging out a bit.

![image](https://github.com/user-attachments/assets/7e6b5d29-1720-490d-9403-f3c648a1df14)

**Comment by BreakBB** [comment]
> Does this happen when two objective icons are stacked exactly at the same location, such as in a dungeon?

Yes, this happens when multiple icons with glow are overlapping.

> This behavior seems relatively new. Have there been any changes that might have caused it?

We have yet to find out if this is new or if it has "always" been like this.

Currently I think, a fix would be to to not draw "the same icon" over and over again at the same location (for example if you need multiple items from the same dungeon, then only a single icon would be drawn. But if you need an item and also slay an NPC, then a loot bag and a sword should show).

**Comment by Gogo1951** [comment]
Yeah it's weird... inconsistent.

Here today, multiple quests / objectives and it's not causing the solid block.

<img width="1390" alt="image" src="https://github.com/user-attachments/assets/81b3c1bf-4739-49e5-8f4e-5e9a9cf11a17" />

<img width="423" alt="image" src="https://github.com/user-attachments/assets/46af36e2-b3a1-4e18-b247-baccb4c71545" />

**Comment by Gogo1951** [comment]
<img width="270" alt="image" src="https://github.com/user-attachments/assets/e3082a9e-5263-4530-ba8e-8c7607df089d" />

Doesn't seem to impact mob icons for multiple quest objectives. Only dungeon icons. Curious.

**Comment by Gogo1951** [comment]
Can't consistently reproduce this.
